src/redux/reducer.js

```diff
@@ -44,4 +47,6 @@ export const reducer = createReducer(initialState, {
     [REMOVE_VOLTAGE_LEVEL_DIAGRAM]: (state, action) => {
         state.diagram = null;
     }
-});
\ No newline at end of file
+});
+
+export const store = createStore(reducer);
```

Comment:
Whyt have you move store creation here. This is strange to put store creation in reducer file.

Reply:
ok i'll move it in a seprate file but it should be in the redux folder. what do you think ?

Reply:
ok